public/locales/en/translation.json

```
@@ -529,8 +529,11 @@
     "assignedTo": "Assigned To",
     "assignTo": "Assign To",
     "volunteer": "Volunteer",
+    "volunteered": "Volunteered",
     "volunteerGroup": "Volunteer Group",
+    "volunteerGroups": "Volunteer Groups",
```

Biome (2.1.2)

[error] The key volunteerGroups was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)